src/sxapi/publicV2/todos.py

```diff
@@ -0,0 +1,114 @@
+class Todos:
```

Comment:
this should be singular: `Todo`

it's mainly there to update, post, delete a single Todo

Reply:
If we're just talking that this is the wrapper for the API, and there will be `Todo` class that transparently handles updates, I'm ok with the naming.